docs/dev/connectors/kafka.zh.md

```diff
@@ -88,7 +88,7 @@ For most users, the `FlinkKafkaConsumer08` (part of `flink-connector-kafka`) is
         <td>>= 1.0.0</td>
         <td>
         This universal Kafka connector attempts to track the latest version of the Kafka client.
-        The version of the client it uses may change between Flink releases.
+        The version of the client it uses may change between Flink releases. As of this release, it uses the Kafka 2.2.0 client.
```

Comment:
I think we should not reference `As of this release`. In 6 months nobody knows what this refers to.
Could you replace that with the correct version (presumably 1.8) in both languages?

Reply:
This sentence was introduced in FLINK-12024 which affects only 1.9. 

What about change this to `Starting with Flink 1.9 release, it uses...` ?

Reply:
Perfect. Thank you!